test/test_s3_utils.py

```diff
@@ -43,7 +86,12 @@ def test_stg(ff_staging_envname):
     test_stg(stg_beanstalk_env)
 
 
+@pytest.mark.integrated
 def test_s3utils_creation_ff_prd():
+    # TODO: I'm not sure what this is testing, so it's hard to rewrite
+    #   But I fear this use of env 'data' implies the GA test environment has overbroad privilege.
+    #   We should make this work without access to 'data'.
+    #   -kmp 13-Jan-2021
```

Comment:
Doing the exact same thing as the above test, except with production. The "creation" part here is the resolution of the S3 buckets.